Makefile

```diff
@@ -1,27 +1,28 @@
 .PHONY: test
 deps:
-	npm i -g jshint istanbul mocha
-	npm i
+	@npm install -g eslint mocha istanbul
+	@npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-import eslint-plugin-react
+	@npm i
 lint:
-	jshint .
+	@eslint .
 test:
-	make lint
-	make cover
+	@make lint
+	@make cover
```

Comment:
Tests commands should not be silent (only sensitive commands containing password, user info, etc.)